.github/workflows/build-container.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+      
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [make-date-tag]
+    if: startsWith(github.repository, 'opensciencegrid/')
+    strategy:
+      fail-fast: False
+      matrix:
+        repo: ['latest']
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Generate tag list
+      id: generate-tag-list
+      env:
+        REPO: ${{ matrix.repo }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
+      run: |
+        #docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
+        docker_repo=opensciencegrid/comanage-utils
+        tag_list=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          for image_tag in "$REPO" "$REPO-$TIMESTAMP"; do
```

Comment:
```suggestion
          for image_tag in "latest" "latest-$TIMESTAMP"; do
```

Reply:
Fine but it was one less thing to hard code with the matrix